### DISCUSSION TITLE
react-chartjs-2 compability

### DISCUSSION THREAD

**Discussion by totaldekadens** [diegomura/react-pdf#2422] [Q&A]
Hi! 

I'm very new using this package so I need help understanding if it's possible to add my own customized components into the pdf. Is it possible to apply a chart from react-chartjs-2 for example? I want the pdf to be as similiar as possible what is showing on my page.

**Comment by Yukigamine** [comment]
Did you ever figure out the answer to this question? I'm looking into this package for the first time and trying to determine the same.

Can I just drop an existing custom component into the document and tweak how it renders as-needed?
Or are we forced to use the supported primitives?

**Comment by crobinson42** [comment]
related thread: https://github.com/diegomura/react-pdf/discussions/1784#discussioncomment-13181690
